docs/sphinx/source/user_guide/bifacial.rst

```diff
@@ -24,7 +24,7 @@ irradiance:
 
 
 pvfactors
-=========
+---------
```

Comment:
These were being incorrectly interpreted as top-level headers.  This demotes them to second-level.